catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -990,6 +991,11 @@ private boolean isChangeTypeDate(AttributeChange attributeChange, Metacard resul
         .equals(AttributeType.AttributeFormat.DATE);
   }
 
+  private boolean isShareableMetacard(Metacard inputMetacard) {
```

Comment:
❗️ If this is truly generic, then all metacards are potentially shareable, just like with `security.access-individuals|groups`.